ddtrace/internal/writer/writer.py

```diff
@@ -451,7 +451,6 @@ class AgentWriter(HTTPWriter):
     def __init__(
         self,
         agent_url,  # type: str
-        priority_sampling=False,  # type: bool
```

Comment:
The `priority_sampling` parameter is never used. Since the AgentWriter is an internal component, it is safe to remove this argument.